ui/components/config/mcp-client-form.tsx

```
@@ -1,342 +1,342 @@
-			...prev,
-			stdio_config: {
-				...prev.stdio_config!,
-				[field]: value,
-			},
-		}));
-	};
+  const handleStdioConfigChange = (field: keyof MCPStdioConfig, value: string | string[]) => {
+    setForm((prev) => ({
+      ...prev,
+      stdio_config: {
+        ...prev.stdio_config!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)